pkg/services/ngalert/notifier/channels/threema.go

```diff
@@ -127,6 +130,22 @@ func (tn *ThreemaNotifier) Notify(ctx context.Context, as ...*types.Alert) (bool
 		tmpl(`{{ template "default.message" . }}`),
 		path.Join(tn.tmpl.ExternalURL.String(), "/alerting/list"),
 	)
+
+	var imageURL string
+	_ = withStoredImages(ctx, tn.log, tn.images,
+		func(index int, image *ngmodels.Image) error {
+			// If there is an image for this alert and the image has been uploaded
+			// to a public URL then add it to the request. We cannot add more than
```

Comment:
I think that we could make changes here for threema - we just attach the image to the already templated message, so there's no reason we couldn't attach every image we find to the message.

Reply:
Do you have a link to their docs for images? I cannot find much when it comes to message formats.

Reply:
The "Image" thing is just part of the message text. I don't think this is something we should block on.